package.json

```diff
@@ -32,9 +32,9 @@
     "@tiptap/suggestion": "^3.0.0",
     "@tiptap/vue-3": "^3.0.0",
     "@uppy/core": "^4.3.1",
-    "@uppy/drag-drop": "^4.0.5",
-    "@uppy/progress-bar": "^4.1.0",
-    "@uppy/tus": "^3.0.1",
+    "@uppy/drag-drop": "^4.1.2",
```

Comment:
Could we also upgrade `@uppy/core` to v4.4.4? (that is what `@uppy/drag-drop` and other dependencies request)

Reply:
sure f4395524fa4edb4776ac5d5dcd00268c098f4f6d